src/components/ebay-chip/index.marko

```diff
@@ -0,0 +1,19 @@
+$ const textId = component.elId("text");
+
+<span class="chip">
```

Comment:
We should allow spread for probably the root element.

Reply:
Added